solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/org-policies/exceptions/compute-restrict-vpc-peering-except-host-project.yaml.yaml

```diff
@@ -0,0 +1,50 @@
+# # Copyright 2021 Google LLC
+# #
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# # you may not use this file except in compliance with the License.
+# # You may obtain a copy of the License at
+# #
+# #      http://www.apache.org/licenses/LICENSE-2.0
+# #
+# # Unless required by applicable law or agreed to in writing, software
+# # distributed under the License is distributed on an "AS IS" BASIS,
+# # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# # See the License for the specific language governing permissions and
+# # limitations under the License.
+# #########
+# #
+# # GCP Organization Policies
+# # Org policies that correspond with a Guardrail will contain a label indicating what Guardrails it helps in enforcing
+# # https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints
+# #
+# # Constraint: constraints/compute.restrictVpcPeering
+# #
+# # This list constraint defines the set of VPC networks
+# # that are allowed to be peered with the VPC networks belonging to this project, folder, or organization.
+# # The allowed/denied list of networks must be identified in the form:
+# # under:organizations/ORGANIZATION_ID, under:folders/FOLDER_ID, under:projects/PROJECT_ID,
+# # or projects/PROJECT_ID/global/networks/NETWORK_NAME.
+# #
+# # This exception is for the VPC in the host project. It allows VPCs from a defined list to be peered with
+# # the VPC in the host project.
+# #
+# #########
+# apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+# kind: ResourceManagerPolicy
+# metadata:
+#   name: compute-restrict-vpc-peering-except-hub-project
```

Comment:
since the `policies` namespace is currently shared by all clients, it would help avoid duplicate names if the host-project-id is included in the metadata.name
i.e.
`name: compute-restrict-vpc-peering-except-host-project-id # kpt-set: compute-restrict-vpc-peering-except-${host-project-id}`

Reply:
Would it be easier to create a client-policies namespace so they clients can have some additional autonomy and reduce risks of naming collisions?

Reply:
yes, it should eventually get created.
I think the "Organization Policy Administrator" role can only be granted at the org level, so we would need to find a way to limit the scope of each client to their client-folder.
maybe this could be done with IAM conditions?

Reply:
I haven't looked at conditions yet but that might be an option. Forgot you couldn't assign the Org admin at the folder level.

Reply:
thank you alain, I will add the host-project-id to the metadata name

Reply:
I think we should keep the policies namespace for now until someone can validate how we can limit the scope of a client-name-policies namespace with Organization Policy Administrator role to the client folder.

Reply:
@richtremblay-ssc @jasoncheff-ssc please add this feature to our backlog. 
validate how we can limit the scope of a client-name-policies namespace with Organization Policy Administrator role to the client folder. Then, modify the file client-landing-zone/....host-project/org-policies/exceptions/compute-restrict-vpc-peering-except-host-project.yaml to use the new namespace.